src/containers/shared/components/Transaction/AMM/AMMBid/parser.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  getAMMAccountID,
+  getAuthAccounts,
+  getMinBid,
+  getMaxBid,
+} from '../../../../metaParser'
+
+export function parser(tx: any, meta: any) {
+  const accountID = getAMMAccountID(meta)
```

Comment:
could this be called `ammAccountID` instead or something? So it's not confused with the account that sent the tx

Reply:
its clear in the context of the file. Gonna leave it as is.

Reply:
It's not clear in the context of `Simple.tsx`, where this is used

Reply:
I changed the labels to all say "AMM Account ID" and changed the name of the variable to ammAccountID